docs/content.zh/docs/dev/datastream/fault-tolerance/state.md

```diff
@@ -561,6 +561,17 @@ val counts: DataStream[(String, Int)] = stream
     })
 ```
 
+### 跟踪 keyed state 的访问延迟
+
+Flink also allows to track the keyed state access latency for all state-backends. This feature is disabled by default.
+To enable this feature you must set the `state.backend.latency-track.keyed-state-enabled` as true in the [Flink configuration]({{< ref "docs/deployment/config" >}}#state-backends-latency-tracking-options).
+
+Once tracking keyed state access latency is enabled, Flink will sample the state access latency every `N` access, in which `N` is defined by `state.backend.latency-track.sample-interval`.
+This configuration has the default value as 128, and a smaller value will get more accurate result but higher performance impact as sampled more frequently.
```

Comment:
```suggestion
This configuration has a default value of 128. A smaller value will get more accurate results but have a higher performance impact since it is sampled more frequently.
```

Reply:
**same for English version**